PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,8 +4,10 @@ Brief description of the PR. [e.g. Added `null` check on `object` to avoid `Null
 
 **Related Issue:** This PR fixes/closes {issue number}
 
-**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped.
+**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped
 
 **Screenshots:** If applicable, add screenshots to help explain your solution
 
+**Tests required of the reviewer**: Optional description of the tests that the reviewer is asked to do, in addition to the static revision of the code. These tests should have already been done by the committer, so please limit the requests
```

Comment:
I would add just the tests section, not explicitly the "required by the reviewer". In that way who opens the PR can report the results of the tests he made, and such information could be useful to the reviewer even if he is not requested to do them. We can keep a note that says that those tests can be asked to the reviewer.

Reply:
Maybe something like:

**Manual tests**: Optional description of the tests performed to check correct functioning of changes, useful for an efficient review

So nothing is said here about asking the reviewer for tests